src/common/src/buffer/bitmap.rs

```diff
@@ -763,6 +803,16 @@ impl<'a> iter::Iterator for BitmapOnesIter<'a> {
             }
         }
     }
+
+    fn size_hint(&self) -> (usize, Option<usize>) {
+        match self {
+            BitmapOnesIter::Range { start, end } => {
```

Comment:
Is this only used in all-zero and all-one cases?

Reply:
Yes. I added this function because arrow `BooleanArray::from_iter` requires a upper bound of the iterator.